x-pack/plugins/observability/public/utils/calculate_bucket_size.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { getBucketSize } from './get_bucket_size';
+
+export function calculateBucketSize({ start, end }: { start?: number; end?: number }) {
+  if (start && end) {
```

Comment:
This might be too forgiving. Maybe we should throw an error if it's not the case, or make start & end required parameters.